nuxeo-ai-model/src/main/java/org/nuxeo/ai/model/AiDocumentTypeConstants.java

```diff
@@ -19,40 +19,40 @@
 package org.nuxeo.ai.model;
 
 /**
- * Simple class to store constants related to AI Document Types: Ai_Model and Ai_Corpus
+ * Simple class to store constants related to AI Document Types: Ai_Model and DatasetExport
  */
 public class AiDocumentTypeConstants {
 
     // corpus type constants
-    public static final String CORPUS_TYPE = "AI_Corpus";
+    public static final String DATASET_EXPORT_TYPE = "DatasetExport";
 
-    public static final String CORPUS_SCHEMA = "ai_corpus";
```

Comment:
If this is used for cloud upload then it shouldn't be changed.

Reply:
it shouldn't be. I created this schema.